velox/common/tests/TraceContextTest.cpp

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/common/process/TraceContext.h"
+#include <fmt/format.h>
+#include <gtest/gtest.h>
+#include <thread>
+
+using namespace facebook::velox::process;
+
+TEST(TraceContextTest, basic) {
```

Comment:
This test doesn't assert anything, e.g. it won't fail if TraceContext stops working properly. Would it be possible to add a method that returns a map of TraceData and assert on it having the right content?